arlas-core/src/main/java/io/arlas/server/app/Documentation.java

```diff
@@ -64,7 +64,7 @@
             "\n \n" +
             "       :lt:     | {fieldName}  is less than {value}                | numeric " +
             "\n \n" +
-            "       :range:  | {fieldName} is between `{comma separated [min<max] values}`. **OR** operation is applied for the specified ranges | numeric or strings. If the field's type is date, then min & max should be timestamps in millisecond ";
+            "       :range:  | {fieldName} is between `{comma separated [min<max] values}`. **OR** operation is applied for the specified ranges | numeric or strings. If the field's type is date, then min & max should be timestamps in millisecond or a Date Math expression";
```

Comment:
Since `Date Math Expression` is not an elasticsearch terminology and since it's not really explicit, you should not use it here. You should find an Arlas terminology for that and perhaps give some examples.

Reply:
Perhaps something like `date descriptor` or `semantic date`